packages/node_modules/nav-frontend-skjema/src/checkbox.tsx

```diff
@@ -1,20 +1,28 @@
-import PT from 'prop-types';
-/* eslint-disable react/prefer-stateless-function  */
-import React, { Component } from 'react';
+import * as PT from 'prop-types';
+import * as React from 'react';
 import { guid } from 'nav-frontend-js-utils';
-import 'nav-frontend-skjema-style'; // eslint-disable-line import/extensions
-import classNames from 'classnames';
-import SkjemaelementFeilmelding, { skjemaelementFeilmeldingShape } from './skjemaelement-feilmelding';
+import * as classNames from 'classnames';
+import SkjemaelementFeilmelding, {SkjemaelementFeil, skjemaelementFeilmeldingShape} from './skjemaelement-feilmelding';
+
+import 'nav-frontend-skjema-style';
```

Comment:
npm-imports burde være ovenfor relative import.
Bygget har gått ok, så mistenker at linteren er litt funky her nå. :/

Reply:
Fikset. Fra tidligere har jeg vært vant med at man skiller på import av js/ts og styling, så hadde flyttet import'en over alt 😔
Fikk ingen linte feil da jeg konverterte, så vet ikke om det er satt opp for typescript?